docs/en/observability/quickstarts/monitor-k8s-otel.asciidoc

```diff
@@ -0,0 +1,65 @@
+
+All the components will be deployed through the https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack[opentelemetry-kube-stack] helm chart. They include:
+
+* https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator]
+* `DaemonSet` EDOT Collector configured for node level metrics.
+* `Deployment` EDOT Collector configured for cluster level metrics.
+* Instrumentation object for applications https://opentelemetry.io/docs/kubernetes/operator/automatic/[auto-instrumentation].
+
+For a more detailed description of the components and advanced configuration refer to the https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md[elastic/opentelemetry] GitHub repository.
+
+[discrete]
+== Prerequisites
+
+* An {es} cluster, self-managed or a hosted {ess} deployment on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data.
+* A running Kubernetes cluster (v1.23 or newer).
+* https://kubernetes.io/docs/reference/kubectl/[Kubectl].
+* https://helm.sh/docs/intro/install/[Helm].
+* (optional) https://cert-manager.io/docs/installation/[Cert-manager], if you opt for automatic generation and renewal of publicly trusted TLS certificates.
+
+[discrete]
+== Collect your data
+
+.  In {kib}, go to **Observability** and click **Add Data**.
+
+. Under the question *`What do you want to monitor?`* select **Kubernetes**, and then select the **OpenTelemetry: Full Observability** option.
++
+[role="screenshot"]
+image::images/quickstart-k8s-otel-entry-point.png[Kubernetes-OTel entry point]
+
+. Follow the on-screen instructions to perform the installation of all needed components.
++
+[NOTE]
+====
+The default installation deploys the OpenTelemetry Operator with a self-signed TLS certificate. For automatic certificate renewal with https://cert-manager.io/docs/installation/[cert-manager], refer to https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md#cert-manager[cert-manager integrated installation] for instructions on customizing the `values.yaml` file before running the `helm install` command.
+====
++
+Install the OpenTelemetry Operator using the kube-stack Helm chart and the provided values file. You will run a few commands to:
++ 
+ * Add the helm chart repository needed for the installation.
+ * Create a namespace.
+ * Create a secret with an API Key and the {es} endpoint to be used by the collectors.
+ * Deploy the `opentelemetry-kube-stack` helm chart with the provided `values.yaml`.
+
+[discrete]
+== Visualize your data
+
+After installation is complete and all relevant data is flowing into Elastic,
+the **Visualize your data** section provides a link to the dashboard used to monitor the health of the cluster (*[OTEL][Metrics Kubernetes]Cluster Overview*).
```

Comment:
I would move this out of parenthesis by using the dashboard name in the sentence:

After installation is complete and all relevant data is flowing into Elastic,
the **Visualize your data** section provides a link to the *[OTEL][Metrics Kubernetes]Cluster Overview* dashboard used to monitor the health of the cluster.

Reply:
The reason of adding it in the parenthesis at the end of the sentence was to reduce the importance of the exact name of the dashboard, as I'm expecting this to change soon or extra dashboards and assets added.

But I'm ok with the change, it looks better!